main.tf

```diff
@@ -22,6 +22,9 @@ locals {
   # Determine if auto scaling is enabled
   auto_scaling_enabled = var.auto_scaling == null ? [] : [1]
 
+  # Determine if host_flavor is used
+  host_flavor_set = var.member_host_flavor != null
```

Comment:
I'm guess you meant to use ternary syntax here, but this logic is invalid. Your IDE or pre commit not catch this?

Reply:
It did not, and the terraform plans I ran worked as intended. Should I just use that condition instead of the variable?

Reply:
Does pre-commit fail for you locally? Pipeline is failing with this:
```
Terraform validate.......................................................Failed
- hook id: terraform_validate
- exit code: 1

Command 'terraform init' successfully done: examples/fscloud
Command 'terraform init' successfully done: examples/backup
Command 'terraform init' successfully done: examples/basic
Validation failed: examples/basic
{
  "format_version": "1.0",
  "valid": false,
  "error_count": 1,
  "warning_count": 0,
  "diagnostics": [
    {
      "severity": "error",
      "summary": "Unsupported block type",
      "detail": "Blocks of type \"host_flavor\" are not expected here.",
      "range": {
        "filename": "../../main.tf",
        "start": {
          "line": 183,
          "column": 5,
          "byte": [93](https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/actions/runs/8251237578/job/22567841073#step:6:94)45
        },
        "end": {
          "line": 183,
          "column": 16,
          "byte": 9356
        }
      },
      "snippet": {
        "context": "resource \"ibm_database\" \"postgresql_db_host_flavor\"",
        "code": "    host_flavor {",
        "start_line": 183,
        "highlight_start_offset": 4,
        "highlight_end_offset": 15,
        "values": []
      }
    }
  ]
}
```

I actually don't think thats related to this line of code, but I was expecting something like this:
`host_flavor_set = var.member_host_flavor != null ? true : false`